src/main/java/com/powsybl/openloadflow/equations/JacobianMatrix.java

```diff
@@ -185,8 +186,9 @@ public Matrix getMatrix() {
     }
 
     private LUDecomposition getLUDecomposition() {
+        Matrix matrix = getMatrix();
```

Comment:
I think this is not needed anymore

Reply:
it is, this is a serious bug that was hidden by a log that was calling luckily getMatrix before computation

Reply:
ok, my bad!